src/components/Item/Item.jsx

```diff
@@ -0,0 +1,80 @@
+import { useState } from 'react';
+import { useItems } from '../../context/ItemsContext';
+import styles from './Item.css';
+
+export default function Item({ item }) {
+  const [updateItem, setUpdateItem] = useState(item.item);
+  const [updateEmoji, setUpdateEmoji] = useState(item.emoji);
+  const [editing, setEditing] = useState(false);
+  const { handleEditItem, handleDeleteItem } = useItems();
+
+  let itemContent;
+
+  if (editing) {
+    itemContent = (
+      <form className={styles.editForm}>
+        <input
+          value={updateItem}
+          onChange={(e) => {
+            setUpdateItem(e.target.value);
+          }}
+          aria-label={`Editing ${item.item}`}
+          className={styles.header}
+        />
+        <input
+          value={updateEmoji}
+          onChange={(e) => {
+            setUpdateEmoji(e.target.value);
+          }}
+          aria-label={`Editing ${item.emoji}`}
+          className={styles.header}
+        />
+
+        <button
+          aria-label={`Save changes to ${item.item}`}
+          onClick={(e) => {
```

Comment:
The `onClick` handlers shouldn't need an `e.preventDefault` just the `onSubmit` handler

Reply:
Removed; thank you!